src/components/StatesButton/StatesButton.tsx

```diff
@@ -0,0 +1,72 @@
+import * as React from 'react';
+import * as classNames from 'classnames';
+import style from './StatesButton.st.css';
+import { Button, SIZE } from '../Button';
+import { Check } from '../../icons/dist';
```

Comment:
I think we'll work on icons in a different PR.
Let's start by using a local adjacent file, that won't be public.
So we can also remove the svg build process.

Reply:
are you sure? @shlomitc told be to use `svg2react-icon`

Reply:
yeah, it's just a bit of a complication for this pr.
We can do this on another branch.
And maybe we'll be able to upgrade to Yoshi 4, and use the svg import feature.